website/src/components/FlaggableElement.tsx

```diff
@@ -69,11 +70,15 @@ export const FlaggableElement = (props) => {
     >
       <Grid templateColumns="1fr min-content" gap={2}>
         <PopoverAnchor>{props.children}</PopoverAnchor>
-        <PopoverTrigger>
-          <Button h="full">
-            <FlagIcon className="w-4 text-gray-400 group-hover:text-gray-500" aria-hidden="true" />
-          </Button>
-        </PopoverTrigger>
+        <Tooltip hasArrow label="Report" bg="red.600">
+          <div>
```

Comment:
is this div necessary?

Reply:
Yeah. The Tooltip component refuses to work if its child element is a PopoverTrigger. Same thing in reverse. Tried several approaches but only this one worked.

Reply:
I concur, the Chakra docs show using `Box` which is basically `div`.